ui/intermittent-failures/GraphAlternateView.jsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable array-callback-return */
+
+import React from 'react';
+import ReactTable from 'react-table';
+
+const GraphAlternateView = ({ graphData, colNum }) => {
```

Comment:
Since `colNum` is only designed to take one or two columns, perhaps this could be changed so it defaults to 1 and there's only two if someone uses the prop. Otherwise if someone enters a number greater than 2 it'll fail silently.